bugzilla/harmony
# renewed attempt at dockerization

TODO:

- lots of stuff

To test, you can:

- run Makefile.PL
-  `docker build . -t bugzilla`
- `docker run --rm -e BZ_db_host=DATABASE -e BZ_urlbase=http://localhost:3000/ -e BZ_db_pass=bugs -ti bugzilla checksetup`
- `docker run --rm -e BZ_db_host=DATABASE -e BZ_urlbase=http://localhost:3000/ -e BZ_db_pass=bugs -ti bugzilla daemon` -- this one should result in a running server.
